### ISSUE TITLE
populate fields when entity creation

### ISSUE THREAD

**Issue by gabrielmdc** [gabrielmdc/greenpi-backend#73]
Example (from measure-repository)
FROM
```typescript
create(document: IMeasure): Promise<IMeasure> {
        return MeasureModel.create(document)
        .then(rejectIfNull('Measure not found'))
        .then(toObject)
        .then(normalizeFiledNames);
    }
```
TO
```typescript
create(document: IMeasure): Promise<IMeasure> {
        return MeasureModel.create(document)
        .then(rejectIfNull('Measure not found'))
        .then((o: IMeasureModel) => o.populate({path:'sensor', populate: {
            path: 'type'
        }}))
        .then(toObject)
        .then(normalizeFiledNames);
    }
```

**Comment by gabrielmdc** [comment]
I would use this: (the the code from above does not work)
```typescript
.then((o: IMeasureModel) => MeasureModel.populate(o, {path: 'sensor', populate: {
            path: 'type'
        }}))
```

**Comment by gabrielmdc** [comment]
Fixed in #75
